linodecli/configuration/__init__.py

```diff
@@ -304,21 +304,20 @@ def configure(
 
         if ENV_TOKEN_NAME in os.environ:
             print(
-                f"""Using token from {ENV_TOKEN_NAME}.
-Note that no token will be saved in your configuration file.
-    * If you lose or remove {ENV_TOKEN_NAME}.
-    * All profiles will use {ENV_TOKEN_NAME}."""
+                f"Using token from {ENV_TOKEN_NAME}.\n"
+                "Note that no token will be saved in your configuration file.\n"
+                f"    * If you lose or remove {ENV_TOKEN_NAME}.\n"
+                f"    * All profiles will use {ENV_TOKEN_NAME}.\n"
```

Comment:
Just to keep things consistent, I think we can drop the last newline here 👍 

```suggestion
               f"    * All profiles will use {ENV_TOKEN_NAME}."
```